spec/labor/sidekiq/worker_retries_exhausted_reporter_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+describe Sidekiq::WorkerRetriesExhaustedReporter, type: :labor do
+  it "increments worker retries exhausted in Datadog" do
+    allow(DataDogStatsClient).to receive(:increment)
+    described_class.report_final_failure("class" => "TempWorker")
```

Comment:
```suggestion
    described_class.report_final_failure(class: TempWorker)
```
do you know if this new syntax would actually work? i feel like the hashrocket syntax is outdated, so i wonder if we could use this one instead?

Reply:
The reason I made this a string is bc that is how the actual arguments will be passed to the middleware. All job arguments are serialized when using Sidekiq so they will be strings when we parse them.